src/bindings/python/tests/test_graph/test_core.py

```diff
@@ -127,8 +127,8 @@ def test_partial_shape():
     assert list(ps.get_max_shape()) == [1, 2, 3]
     assert list(ps.get_min_shape()) == [1, 2, 3]
     assert list(ps.to_shape()) == [1, 2, 3]
-    assert repr(shape) == "<Shape: {1, 2, 3}>"
-    assert repr(ps) == "<PartialShape: {1,2,3}>"
+    assert repr(shape) == "<Shape: [1, 2, 3]>"
+    assert repr(ps) == "<PartialShape: [1,2,3]>"
```

Comment:
Why `Shape` with white-spaces and `PartialShape` has no them?

Reply:
Yes, it looks misaligned. Removed extra spaces from Shape, to align with PartialShape.